test/browser_tests/page_objects/page_the-bureau.js

```diff
@@ -6,6 +6,37 @@ function TheBureauPage() {
   };
 
   this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.hero = element( by.css( '.hero__bureau' ) );
+
+  this.heroVideoPlayer = this.hero.element( by.css( '.video-player') );
+
+  this.bureauHistory = element( by.css( '.bureau-history' ) );
+
+  this.bureauFunctions = element( by.css( '.bureau-functions' ) );
+
+  this.bureauMission = element.all(
+                        by.css( '.js-mobile-carousel .media-stack_item' )
```

Comment:
I've fixed the linting issues but unsure why it would cause `nav-primary.spec` to timeout.